src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4ActivityPanel/DownloadActivityModal.tsx

```diff
@@ -0,0 +1,103 @@
+import { DownloadOutlined } from '@ant-design/icons'
+import { t, Trans } from '@lingui/macro'
+import { Button, Modal, ModalProps } from 'antd'
+import InputAccessoryButton from 'components/buttons/InputAccessoryButton'
+import FormattedNumberInput from 'components/inputs/FormattedNumberInput'
+import { useBlockNumber } from 'hooks/useBlockNumber'
```

Comment:
Lets use the wagmi hook for this: https://wagmi.sh/react/api/hooks/useBlockNumber#useblocknumber

Reply:
Done